server/views/partials/header.njk

```diff
@@ -15,6 +15,15 @@
 
     <nav aria-label="Account navigation">
       <ul class="hmpps-header__navigation">
+        {% if caseLoads.length > 1 %}
+          <li class="hmpps-header__navigation__item">
+            <img class="hmpps-header__link__icon" src="/assets/images/location.svg" aria-hidden="true" alt="" />
+            <a data-qa="changeCaseLoad" class="hmpps-header__link" href="{{ changeCaseLoadLink }}" data-test="manage-account-link" target="_blank" rel="noopener noreferrer">
```

Comment:
Will using `noreferrer` cause issues for the back link?
We use `referrerpolicy="no-referrer-when-downgrade"` for the change caseload link in the prisoner-profile

Reply:
Hadn't considered actually, will update on my next pr